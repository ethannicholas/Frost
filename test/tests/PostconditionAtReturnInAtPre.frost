class PostconditionAtReturnInAtPre {
    @post(@return = @pre(@return))
    method test(x:Int):Int {
        return x * x
    }
}
