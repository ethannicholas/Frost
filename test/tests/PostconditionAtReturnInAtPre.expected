PostconditionAtReturnInAtPre.panda:2:26: error: cannot use @return inside @pre
1 error
